packages/cli/src/cli-default-servient.ts

```diff
@@ -28,6 +28,7 @@ import { HttpsClientFactory } from "@node-wot/binding-http";
 import { CoapClientFactory } from "@node-wot/binding-coap";
 import { CoapsClientFactory } from "@node-wot/binding-coap";
 import { MqttClientFactory }  from "@node-wot/binding-mqtt";
+import { MBusClientFactory }  from "@node-wot/binding-mbus";
```

Comment:
Sorry, but I would not include this binding into the default servient. We don't have defined a clear policy yet but as you can see a lot of other bindings are actually missing. Taking into account the fact that this binding is not fully implemented I would prefer to leave it out.